src/redisenterprise/azext_redisenterprise/azext_metadata.json

```diff
@@ -0,0 +1,3 @@
+{
+    "azext.minCliCoreVersion": "2.15.0"
```

Comment:
for first on-board extension, we will recommend to mark as `Preview` for one sprint to avoid potential changes for command interface. Without this tag, any breaking change is not expected in the future except.

Reply:
Fixed